ciongke/test-image/Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM ubuntu:16.04
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
```

Comment:
The docker styleguide explicitly asks you to minimize the number of cache layers:

https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/#/minimize-the-number-of-layers